md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/ProcessAPI.java

```diff
@@ -708,14 +783,13 @@ RestWrapper importData(@ModelAttribute("fileString")
                         daoProcessTemplate.setProcessTemplateId(process.getProcessTemplateId());
                         updateDaoProcess.setProcessTemplate(daoProcessTemplate);
                     }
-                    if (process.getParentProcessId()!=null){
-                    if (dbParentProcess!= null) {
-                        updateDaoProcess.setProcess(dbParentProcess);
+                    if (process.getParentProcessId() != null) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS134)